### DISCUSSION TITLE
Blank screen upon initialisation

### DISCUSSION THREAD

**Discussion by rishivinod7** [mozilla/hubs#3990] [Hubs Cloud Questions] [answered]
I've been attempting to run hubs locally and by forking hubs and setting up my own cloud instance, but all of a sudden I can't even run a fresh hubs repo or my old codebase locally. I have no idea why this is happening and not sure whether this is a problem from my side. I would appreciate some help as soon as possible and am wondering if anyone else is facing this issue. I've followed the most basic steps as well; cloned a new repo, npm ci, npm run dev . Here are the errors:

![error](https://user-images.githubusercontent.com/46593967/110037836-de869500-7d36-11eb-854b-d01912e7fe00.JPG)
![err](https://user-images.githubusercontent.com/46593967/110037855-e6ded000-7d36-11eb-8aa6-96c21561f7b5.JPG)

**Comment by johnshaughnessy** [comment, marked as answer]
Clear local storage and refresh the page. The `Write store failed schema validation` means you have something in your store that violates your current version's schema policies. 

When this error occurs, we should probably just clear local storage for you -- this is on our (very long) to do list.
